notebooks/calculate_sensitivity_from_Crab.ipynb

```diff
@@ -0,0 +1,2032 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    # Three off regions for higher energies:
</pre><p>clarify that this is just to compute the OFF. The sensitivity calculation uses the <code>alpha=0.2</code> defined above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
good point, done.